app/scripts/modules/cloudfoundry/src/serverGroup/configure/wizard/sections/artifactSettings.cf.tsx

```diff
@@ -58,43 +59,34 @@ function isTriggerSource(
 }
 
 class ArtifactSettingsImpl extends React.Component<
-  ICloudFoundryCreateServerGroupArtifactSettingsProps &
-    IWizardPageProps &
-    FormikProps<ICloudFoundryCreateServerGroupCommand>,
+  ICloudFoundryCreateServerGroupArtifactSettingsProps,
   ICloudFoundryServerGroupArtifactSettingsState
 > {
   public static get LABEL() {
     return 'Artifact';
   }
 
-  constructor(
-    props: ICloudFoundryCreateServerGroupArtifactSettingsProps &
-      IWizardPageProps &
-      FormikProps<ICloudFoundryCreateServerGroupCommand>,
-  ) {
-    super(props);
-    this.state = {
-      clusters: [],
-      filteredClusters: [],
-      serverGroups: [],
-      regions: [],
-      allCloudFoundryCredentials: undefined,
-    };
-  }
+  public state: ICloudFoundryServerGroupArtifactSettingsState = {
```

Comment:
I prefer omitting the constructor when possible

Reply:
👍